lib/sockets/sub-emitter.js

```diff
@@ -35,7 +35,7 @@ function SubEmitterSocket() {
  * @api private
  */
 
-SubEmitterSocket.prototype.onmessage = function(){
+SubEmitterSocket.prototype.onmessage = function(sock){
```

Comment:
Absolutely. Should I remove the `@param {net.Socket} sock` description as well, or had that better remain since technically `sock` *is* passed to the handler?

It's the same situation on `SubSocket` as well btw -- `sock` is passed in but never used.